content/50.discussion.md

```diff
@@ -198,7 +198,69 @@ As a result, the standard practice of requiring enrollment in a clinical trial i
+While some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and a lack of health insurance [@doi:10.1002/cncr.28483], a major concern is that patients from certain racial and ethnic groups are marginalized even while seeking healthcare.
+In the United States, many patients experience “othering" from physicians and other medical professionals due to their race or other external characteristics such as gender (e.g., [@doi:10.1207/S15327027HC1602_7]).
+Many studies have sought to characterize implicit biases in healthcare providers and whether they affect their perceptions or treatment of patients.
+A systematic review that examined 37 such studies reported that most (31) identified racial and/or ethnic biases in healthcare providers in many different roles, although the evidence about whether these biases translated to different attitudes towards patients was mixed [@doi:10.1016/j.socscimed.2017.05.009], with similar findings reported by a second systematic review [@doi:10.1111/acem.13214].
+However, data about real-world patient outcomes are very limited, with most studies relying on clinical vignette-based exercises [@doi:10.1016/j.socscimed.2017.05.009], and other analyses suggest that physician implicit bias could impact the patient's perception of the negativity/positivity of the interaction regardless of the physician's explicit behavior towards the patient [@doi:10.1016/j.jesp.2009.11.004].
+Because racism is a common factor in both, negative patient experiences with medical professionals are likely to compound other issues of systemic inequality, such as a lack of access to adequate care, a lack of insurance, or increased exposure to SARS-CoV-2 [@doi:10.1542/peds.2020-003657].
+Furthermore, the experience of being othered is not only expected to impact patients' trust in and comfort with their provider, but also may directly impact whether or not the patient is offered the opportunity to participate in a clinical trial at all.
+Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
+For example, researchers utilized a linguistic analysis to assess mean word count of phrases related to clinical trial enrollment, such as voluntary participation, clinical trial, etc. [@doi:10.1002/cncr.28483].
+The data indicated that the mean word count of the entire visit was 1.5 times more for white patients in comparison to Black patients.
+In addition, the greatest disparity between white and Black patients' experience was the discussion of risks, with over 2 times as many risk-related words spoken with white patients than Black patients [@doi:10.1002/cncr.28483].
+Thus, it is likely that COVID-19 clinical trial information is not being equally discussed with Black patients. 
```

Comment:
This might be better framed as: if it's like other trials, then...

The paper above pre-dates COVID-19.

Reply:
```suggestion
The trends observed for other clinical trials raise the concern that COVID-19 clinical trial information may not be discussed as thoroughly or as often with Black patients compared to white patients. 

```